app/Http/Controllers/Base/AccountController.php

```diff
@@ -51,8 +54,20 @@ public function __construct(AlertsMessageBag $alert, AuthManager $authManager, U
      */
     public function index()
     {
+        if (config('oauth2.enabled') == true) {
+            $oauth2_ids = [];
+            foreach (preg_split('~,~', \Auth::user()->getAttribute('oauth2_id')) as $id) {
+                $split = preg_split('~:~', $id);
+                if (! empty($split[1])) {
+                    $oauth2_ids = Arr::add($oauth2_ids, $split[0], $split[1]);
+                }
+            }
+        }
+
         return view('base.account', [
             'languages' => $this->getAvailableLanguages(true),
+            'enabled_providers' => config('oauth2.enabled') == true ? $this->getEnabledProviderSettings() : '',
```

Comment:
Strict comparisons only, but this can also completely skip the boolean checking, its either going to be a truthly value or evaluate to false anyways.